### DISCUSSION TITLE
Mail ayudante de TP

### DISCUSSION THREAD

**Discussion by eGoicoechea** [dds-jv/foro#120] [Consultas generales] [answered]
Hola, Soy Emilia Goicoechea del grupo 16 de los viernes a la noche. 
Queria preguntarles si me pueden pasar el mail de Gus ya que es nuestro ayudante de tp y queremos comunicarnos con el para entregarle el POC que ya tenemos resuelto, pero no tenemos forma de comunicarnos con el. 
Desde ya muchas gracias. 
Saludos

**Comment by gastonprieto** [comment]
Buenas!

Seguramente hoy o en estos días te llega el mail de presentación, si uds lo subieron para hoy no hay drama. Igualmente te paso el mail; crespi.gustavo@gmail.com

**Comment by eGoicoechea** [comment]
Sisi, el tp esta en el repo

**Comment by eGoicoechea** [comment, marked as answer]
Ya se contacto con nosotros. Muchas gracias!
